packages/static-scripts/scripts/build.js

```diff
@@ -1,12 +1,51 @@
 #!/usr/bin/env node
-const spawn = require("react-dev-utils/crossSpawn")
-const gulpArgs = require("./gulpArgs")
 
-// Todo: Extract
+// Here we set the NODE_ENV to whatever was passed in
+// by the calling process, and default to production
+process.env.NODE_ENV = process.env.NODE_ENV 
+    ? process.env.NODE_ENV 
+    : "production"
 
-const scriptCmd = [
-  "cross-env",
-  ["NODE_ENV=production", "gulp", ...gulpArgs, "build"],
-]
+const imagemin = require("./runners/imagemin")
+const log = require("./utils/log")
+const postCss = require("./runners/postcss")
+const purgeCss = require("./runners/purgecss")
+const generator = require("./runners/generator")
+const Orchestrator = require("orchestrator")
+const svgSprite = require("./runners/svg-sprite")
+const webpack = require("./runners/webpack")
 
-spawn.sync(...scriptCmd, { stdio: "inherit" })
+// Here we define the tasks using orchestrator,
+// an extremely lightweight and efficient task executor
+const taskMgr = new Orchestrator();
```

Comment:
I'm a little nervous about `orchestrator` because doesn't seem to be maintained. There hasn't been an update in over a year.

Reply:
I wasn't too worried because TBH Gulp is in the same boat. Everyone uses 3.x because 4.0 has been stale for over a year as well.

People submit PRs if there's a bug and somebody will merge them, but it isn't being improved actively.

Reply:
We could do something like this though: https://medium.com/@_jh3y/how-to-build-your-own-task-runner-with-es6-and-node-85d871a4a0c

Reply:
My thought is that orchestrator is fine for now. We'll consider alternatives if it becomes an issue. That's the beauty of the build process not being API facing.

Reply:
Exactly! 👯‍♂️